mdoc-js/src/main/resources/mdoc.js

```diff
@@ -5,8 +5,8 @@
 
   function loadAll(scope) {
     findDivs().forEach(function(el) {
-      var id = el.getAttribute("id");
-      scope[id](el);
+      var id = el.getAttribute("id").replace("-html-", "_js_");
+      eval(id)(el);
```

Comment:
This has changed because top level exports from SJS are declared with `let` and I don't think there's a way to access them dynamically by name as they're not added to the global scope?